test/utils/secret.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestSecret(t *testing.T, consumerName string) {
```

Comment:
We may wanna name this in a more generic way, such as TestContentInterfaces to allow having both secret seeding and config interfaces inside. 

We can do it once we have tests for the config interface.

Reply:
make sense, I will add this as a TODO comment in the test.